build.sh

```diff
@@ -141,8 +141,12 @@ build_managed_corefx()
 
 build_native_corefx()
 {
+    # Format the Native Code to make sure it is format-compliant
+    echo "Formatting native source code..."
+    "$__nativeroot/format-code.sh"
```

Comment:
Won't this eliminate incremental compilation or is the format script smart enough to only affect the modification time of a file if there is an actual change?